package.json

```diff
@@ -39,10 +39,16 @@
     "vscode-debugprotocol": "^1.11.0"
   },
   "devDependencies": {
-    "fs-extra": "^0.30.0",
-    "tslint": "^3.14.0",
-    "typescript": "1.8.10",
-    "vscode": "^0.11.17"
+    "@types/diff": "0.0.31",
+    "@types/fs-extra": "0.0.35",
+    "@types/lodash": "^4.14.40",
```

Comment:
I notice that travis use an old node v0.10.36 with an old npm v1.4.28. Is this affected? Can we upgrade it to recent version?